network-store-server/src/main/java/com/powsybl/network/store/server/QueryCatalog.java

```diff
@@ -216,6 +218,12 @@ public static String buildCloneIdentifiablesQuery(String tableName, Collection<S
                 "where networkUuid = ? and variantNum = ?";
     }
 
+    public static String buildCloneTemporaryLimitsQuery() {
+        return "insert into temporarylimit(equipmentId, equipmentType, networkUuid, variantNum, side, limitType, name, value, acceptableDuration, fictitious) " +
+                "select equipmentId, equipmentType, ?, ?, side, limitType, name, value, acceptableDuration, fictitious " +
+                "from temporarylimit where networkUuid = ? and variantNum = ?";
```

Comment:
Maybe use a `temporaryLimitTableName` variable, and NETWORK_UUID, VARIANT_NUM... as well ?

Reply:
Did the columns, not the table name (to match what's already there).